demo/demo-springmvc/springmvc-client/src/main/java/org/apache/servicecomb/demo/springmvc/client/SpringmvcClient.java

```diff
@@ -221,17 +222,33 @@ private static void testConfigurationDuplicate() {
 
   private static void testDefaultValues(RestTemplate template, String microserviceName) {
     String prefix = "cse://" + microserviceName;
-
     TestMgr.check("hi test your age is : 20",
-        template.getForObject(prefix + "/default/sayhi",
+        template.getForObject(prefix + "/annotations/sayhi",
             String.class));
 
     TestMgr.check("20",
-        template.getForObject(prefix + "/default/add",
+        template.getForObject(prefix + "/annotations/add",
             String.class));
 
     TestMgr.check("hei test",
-        template.getForObject(prefix + "/default/sayhei",
+        template.getForObject(prefix + "/annotations/sayhei",
             String.class));
   }
+
+  private static void testRequiredBody(RestTemplate template, String microserviceName) {
+    String prefix = "cse://" + microserviceName;
+    Person user = new Person();
+    user.setName("world");
+    TestMgr.check("ha world",
+        template.postForObject(prefix + "/annotations/saysomething?prefix={prefix}",
+            user,
+            String.class,
+            "ha"));
+
+    TestMgr.check("No user data found",
+        template.postForObject(prefix + "/annotations/saysomething?prefix={prefix}",
+            null,
```

Comment:
As mentioned in above comments, "null" may not indicates an empty body. Need to distinguish null or empty.

Reply:
As mentioned in above latest comments, if here can pass "" to user parameter which indicate an empty body? null to indicate a null instance?